core/src/ast/data.rs

```diff
@@ -150,18 +152,35 @@ impl<V: UsesLifetimes, F: UsesLifetimes> UsesLifetimes for Data<V, F> {
 }
 
 /// Equivalent to `syn::Fields`, but replaces the AST element with a generic.
-#[derive(Debug, Clone, PartialEq, Eq)]
+#[derive(Debug, Clone)]
 pub struct Fields<T> {
     pub style: Style,
     pub fields: Vec<T>,
+    pub span: Option<Span>,
```

Comment:
You've convinced me it's better for `Fields` to impl `Spanned` rather than exposing this as a field, but I don't see that trait impl anywhere. Am I missing it?

Reply:
`Spannend` is implemented for any type that implements `ToTokens`, which obtains the `Span` from the returned `TokenStream`.